spec/unit/ssl/certificate_factory_spec.rb

```diff
@@ -20,12 +20,6 @@
   end
 
   describe "when generating the certificate" do
-    it "should return a new X509 certificate" do
-      expect(subject.build(:server, csr, issuer, serial)).not_to eq(
-        subject.build(:server, csr, issuer, serial)
-      )
-    end
-
```

Comment:
I wonder if there is puppet code relying to the fact that previously these were not value objects?

Reply:
@joshcooper is there any good way to find out? Will our acceptance suites uncover any issues?